content/python/concepts/built-in-functions/terms/next/next.md

```diff
@@ -0,0 +1,85 @@
+---
+Title: 'next()'
+Description: 'Returns the next element/value from an iterable object.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Collections'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+## **Goal** :dart:
+
+This entry tries to facilitate your comprehension of the Python `next()` function and how to use it properly to get the next item from an iterator object.
+
+## **Why next() function in Python?**
+
+It is a tool on the iterator that allows you to sequentially go through each element of it, using a common method included in every countable set named `iter()`.
+
+## **You need to know (maybe)**
+
+What is an **iterator**? An object that can contain any set of countable values of objects like [_lists, tuples, dictionaries, sets, strings_], traversing all of them if required.
+
+## **Syntax**
+
+```py
+next(iterator_name, [default_parameter])
+```
+***`Parameters:`*** 
++ iterator_name: name given to the iterator that contain the set of values.
++ default_parameter: [Optional], default value to be printed if end of iterator is reached.
+
+***`Return:`*** 
++ returns next element/value from the iterable object.
+    + _if missing prints the default value_
+    + _if default value is not set, triggers a StopIteration error_
+
+## **Example 1**
+
+This code iterates a list containing four elements, printing each on screen by means of `next()` function.
+
+```codebyte/python
+listems = ['Hi', 27, 'Python', 10] # Here's the initial item's list
+
+listemsIt = iter(lisitems) # First of all, we need to convert list into an iterator
+print(listemsIt) # Shows the object reference of the iterator created
+
+print(next(listemsIt)) # Output on screen: Hi
+
+print(next(listemsIt)) # Output on screen: 27
+
+print(next(listemsIt)) # Output on screen: Python
+
+print(next(listemsIt)) # Output on screen: 10
+```
+
```

Comment:
```suggestion

This outputs the following:

```

And then let's add a \```shell code block that looks like this:

```shell
Hi
27
Python
10
```

Reply:
I can't do nothing to resolve this suggestion because it shows me a message saying that "has been applied or is marked resolved".